driver/src/main/java/org/apache/tinkerpop/gremlin/orientdb/ODBFeatures.java

```diff
@@ -107,7 +108,7 @@ public boolean supportsComputer() {
 
         @Override
         public boolean supportsTransactions() {
-            return false;
+            return transactionalGraph;
```

Comment:
I adopted the OrientGraphNoTx implementation:
https://github.com/orientechnologies/orientdb/blob/master/graphdb/src/main/java/com/tinkerpop/blueprints/impls/orient/OrientGraphNoTx.java#L102